src/app/modules/practicas-academicas/detalle-practica-academica/detalle-practica-academica.component.ts

```diff
@@ -156,8 +156,8 @@ export class DetallePracticaAcademicaComponent {
         .subscribe(
           (res) => {
             const r = <any>res;
-            if (res !== null && r.success !== false) {
-              if (r.status === 200 && res["data"] !== null) {
+            if (res !== null && r.Success !== false) {
+              if (r.Status === 200 && res["data"] !== null) {
                 this.periodos = res["data"]["periodos"];
                 this.proyectos = res["data"]["proyectos"];
                 this.tiposVehiculo = res["data"]["vehiculos"];
```

Comment:
Por ejemplo aquí también faltaría.
```suggestion
              if (r.Status === 200 && res["Data"] !== null) {
                this.periodos = res["Data"]["periodos"];
                this.proyectos = res["Data"]["proyectos"];
                this.tiposVehiculo = res["Data"]["vehiculos"];
```